icd/api/pipeline_compiler.cpp

```diff
@@ -1047,25 +1047,90 @@ void PipelineCompiler::GetPipelineCreationInfoNext(
 // =====================================================================================================================
 VkResult PipelineCompiler::SetPipelineCreationFeedbackInfo(
     const VkPipelineCreationFeedbackCreateInfoEXT* pPipelineCreationFeadbackCreateInfo,
-    const PipelineCreationFeedback*                pPipelineFeedback)
+    const VkGraphicsPipelineCreateInfo*            pCreateInfo,
+    const PipelineCreationFeedback*                pPipelineFeedback,
+    const PipelineCreationFeedback*                pStageFeedback)
 {
-    if (pPipelineCreationFeadbackCreateInfo != nullptr)
-    {
-        if (pPipelineFeedback->feedbackValid)
+  if (pPipelineCreationFeadbackCreateInfo != nullptr)
+  {
+    if (pPipelineFeedback->feedbackValid)
+    {
+      pPipelineCreationFeadbackCreateInfo->pPipelineCreationFeedback->flags |=
+          VK_PIPELINE_CREATION_FEEDBACK_VALID_BIT_EXT;
```

Comment:
FYI: the line length limit is 120 IIRC, so you may be able to fit more stuff on each line where you access array elements if you hoist the reference to that array element as a local reference/pointer.

Reply:
done